include/zephyr/posix/unistd.h

```diff
@@ -55,6 +55,153 @@ pid_t getpid(void);
+#define __z_posix_sysconf_SC_IPV6			  (-1L)
+#define __z_posix_sysconf_SC_JOB_CONTRO			  (-1L)
+#define __z_posix_sysconf_SC_MAPPED_FILES		  (-1L)
+#define __z_posix_sysconf_SC_MEMLOCK			  (-1L)
+#define __z_posix_sysconf_SC_MEMLOCK_RANGE		  (-1L)
+#define __z_posix_sysconf_SC_MEMORY_PROTECTION		  (-1L)
+#define __z_posix_sysconf_SC_MESSAGE_PASSING		  (-1L)
+#define __z_posix_sysconf_SC_MONOTONIC_CLOCK		  (-1L)
+#define __z_posix_sysconf_SC_PRIORITIZED_IO		  (-1L)
+#define __z_posix_sysconf_SC_PRIORITY_SCHEDULING	  (-1L)
+#define __z_posix_sysconf_SC_RAW_SOCKETS		  (-1L)
+#define __z_posix_sysconf_SC_READER_WRITER_LOCKS	  (-1L)
+#define __z_posix_sysconf_SC_REALTIME_SIGNALS		  (-1L)
+#define __z_posix_sysconf_SC_REGEXP			  (-1L)
+#define __z_posix_sysconf_SC_SAVED_IDS			  (-1L)
+#define __z_posix_sysconf_SC_SEMAPHORES			  COND_CODE_1(CONFIG_PTHREAD_IPC,	  \
+								      (SUPPORTED_POSIX1_VERSION), \
+								      (-1L))
+#define __z_posix_sysconf_SC_SHARED_MEMORY_OBJECTS	  (-1L)
+#define __z_posix_sysconf_SC_SHELL			  (-1L)
+#define __z_posix_sysconf_SC_SPAWN			  (-1L)
+#define __z_posix_sysconf_SC_SPIN_LOCKS			  (-1L)
+#define __z_posix_sysconf_SC_SPORADIC_SERVER		  (-1L)
+#define __z_posix_sysconf_SC_SS_REPL_MAX		  (-1L)
+#define __z_posix_sysconf_SC_SYNCHRONIZED_IO		  (-1L)
+#define __z_posix_sysconf_SC_THREAD_ATTR_STACKADDR	  COND_CODE_1(CONFIG_PTHREAD_IPC,	  \
+								      (SUPPORTED_POSIX1_VERSION), \
+								      (-1L))
+#define __z_posix_sysconf_SC_THREAD_ATTR_STACKSIZE	  COND_CODE_1(CONFIG_PTHREAD_IPC,	  \
+								      (SUPPORTED_POSIX1_VERSION), \
+								      (-1L))
+#define __z_posix_sysconf_SC_THREAD_CPUTIME		  (-1L)
+#define __z_posix_sysconf_SC_THREAD_PRIO_INHERIT	  (-1L)
+#define __z_posix_sysconf_SC_THREAD_PRIO_PROTECT	  (-1L)
+#define __z_posix_sysconf_SC_THREAD_PRIORITY_SCHEDULING	  COND_CODE_1(CONFIG_PTHREAD_IPC,	  \
+								      (SUPPORTED_POSIX1_VERSION), \
+								      (-1L))
+#define __z_posix_sysconf_SC_THREAD_PROCESS_SHARED	  (-1L)
+#define __z_posix_sysconf_SC_THREAD_ROBUST_PRIO_INHERIT	  (-1L)
+#define __z_posix_sysconf_SC_THREAD_ROBUST_PRIO_PROTECT	  (-1L)
+#define __z_posix_sysconf_SC_THREAD_SAFE_FUNCTIONS	  (-1L)
+#define __z_posix_sysconf_SC_THREAD_SPORADIC_SERVER	  (-1L)
+#define __z_posix_sysconf_SC_THREADS			  (-1L)
+#define __z_posix_sysconf_SC_TIMEOUTS			  (-1L)
+#define __z_posix_sysconf_SC_TIMERS			  (-1L)
+#define __z_posix_sysconf_SC_TRACE			  (-1L)
+#define __z_posix_sysconf_SC_TRACE_EVENT_FILTER		  (-1L)
+#define __z_posix_sysconf_SC_TRACE_EVENT_NAME_MAX	  (-1L)
+#define __z_posix_sysconf_SC_TRACE_INHERIT		  (-1L)
+#define __z_posix_sysconf_SC_TRACE_LOG			  (-1L)
+#define __z_posix_sysconf_SC_TRACE_NAME_MAX		  (-1L)
+#define __z_posix_sysconf_SC_TRACE_SYS_MAX		  (-1L)
+#define __z_posix_sysconf_SC_TRACE_USER_EVENT_MAX	  (-1L)
+#define __z_posix_sysconf_SC_TYPED_MEMORY_OBJECTS	  (-1L)
+#define __z_posix_sysconf_SC_VERSION			  SUPPORTED_POSIX1_VERSION
```

Comment:
Should be switched to `_POSIX_VERSION` here and elsewhere in (see other comments).

```suggestion
#define __z_posix_sysconf_SC_VERSION			  _POSIX_VERSION
```